philips-hue-adapter.js

```diff
@@ -332,6 +332,25 @@ class PhilipsHueDevice extends Device {
 
           const level = stateToLevel(device.state);
 
+          this.properties.set(
+            'level',
+            new PhilipsHueProperty(
+              this,
+              'level',
+              {
+                '@type': 'BrightnessProperty',
+                label: 'Brightness',
+                type: 'integer',
+                unit: 'percent',
+                minimum: 0,
+                maximum: 100,
+              },
+              level));
+        } else {
+          this['@type'].push('BrightnessProperty');
```

Comment:
This should be removed. `BrightnessProperty` belongs to the property, not the device.